tests/integration/editor/test_editor_views.py

```diff
@@ -125,8 +125,8 @@ def test_get_revisions(log_in_as_cataloger, record_with_two_revisions, api_clien
 
     result = json.loads(response.data)
 
-    assert result[0]['revision_id'] == 1
-    assert result[1]['revision_id'] == 0
+    assert result[0]['revision_id'] == 2
+    assert result[1]['revision_id'] == 1
```

Comment:
Who's creating the extra revision?

Reply:
The download_documents, when there are documents, as I don't want to reimplement the invenio record `create` but just use it, if I do any changes after the `super` runs, I have to create a new revision, any ideas there are welcome.

Reply:
Added a note in the create record method.